README.md

```diff
@@ -10,25 +10,19 @@ pattern is nice, but there is still a lot of room to improve on the ergonomics
 of dealing with remote data. `remote-collection` aims to provide a user-friendly
 experience to the user while keeping 100% type-safety.
```

Comment:
redundant use of the term `user` in `provide a user-friendly experience to the user`. cut it to `...aims to provide a user-friendly experience while keeping 100% type-safety.`